src/main/java/ContactsPage.java

```diff
@@ -39,4 +48,43 @@ public boolean contactsPageIsDisplayed() {
             return false;
         }
     }
+
+
+    public void clickButtonAddNewContact(){
+        try{
+            addNewContact.click();
+        } catch (ElementClickInterceptedException e){
+            e.getStackTrace();
+        }
+
+    }
+
+    public boolean modalWindowIsDisplayed(){
+        return modalWindowAddContact.isDisplayed();
+    }
+
+    public void enterFirstName(String firstNameValue){
+
+        firstNameInputField.sendKeys(firstNameValue);
+    }
+
+    public void enterLastName(String lastNameValue){
+        lastNameInputField.sendKeys(lastNameValue);
+    }
+
+    public void clickSearchInputField(){
```

Comment:
As I have already commented, this is not necessary action.

Reply:
Ok